lib/recurly/hosted-fields.js

```diff
@@ -151,13 +175,15 @@ export class HostedFields extends Emitter {
   fieldConfig (type) {
     const fields = this.config.recurly.fields;
     const field = fields[type];
+    const busGroupId = this.recurly.bus.groupId;
     const { deviceId, sessionId } = this.config;
     const { displayIcon, inputType, selector } = field;
     if (!fields[type]) return;
     return {
       type,
       deviceId,
       sessionId,
+      busGroupId,
```

Comment:
This is the additional configuration value sent to hosted fields to instruct them on which bus group to send their messages